src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -47,13 +47,17 @@ export default class MessagingSessionConfiguration {
   /**
    * Constructs a MessagingSessionConfiguration optionally with userArn, messaging session id, a messaging session
    * endpoint URL, and the chimeClient.
+   *
+   * endpointUrl is deprecated and should not be used. Internally it is resolved on connect via chimeClient if undefined, and
```

Comment:
nit: would suggest mark it deprecated with `@deprecated`  https://code.visualstudio.com/updates/v1_49#_deprecated-tag-support-for-javascript-and-typescript